runtime/caches/lrucache.ts

```diff
@@ -0,0 +1,99 @@
+import { LRUCache } from "npm:lru-cache@10.2.0";
+import {
+  assertCanBeCached,
+  assertNoOptions,
+  baseCache,
+  createBaseCacheStorage,
+} from "./utils.ts";
+
+const MAX_CACHE_SIZE = parseInt(
+  Deno.env.get("MAX_CACHE_SIZE") ?? "1073824",
+); // 1 GB max size of cache
+const TTL_AUTOPURGE = Deno.env.get("TTL_AUTOPURGE") !== "false"; // automatically delete expired items
+const ALLOW_STALE = Deno.env.get("ALLOW_STALE") !== "false"; // automatically allow stale
+const TTL_RESOLUTION = parseInt(
+  Deno.env.get("TTL_RESOLUTION") ?? "30000",
```

Comment:
I think we should prefix with DECO_CACHE_*
```suggestion
  Deno.env.get("TTL_RESOLUTION") ?? "30000",
```